packages/dataviews/package.json

```diff
@@ -1,87 +0,0 @@
-{
```

Comment:
I wonder if we should keep the package, reexport everything from `@wordpress/dataviews` and add a deprecated call (@wordpress/deprecated).

Also after this PR, we should make sure to officially deprecate the package in `npm deprecate` command.

Reply:
Since this is the forked version intended for Calypso's use, I think it's fine to remove the package. What do you think?

Reply:
Yes. I meant we can remove the package and deprecate it from npm.